### PR TITLE
Install script latest release

### PR DESCRIPTION
**Pull request by gtardif**
* got initial script from v0.1.4 release
* updated it to get latest release, and update CLI if it was already installed 
* Added Dockerfile to test it

### CODE REVIEW

File: scripts/Dockerfile-testInstall [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:latest
```
Comment (chris-crone) [inline scripts/Dockerfile-testInstall:1]: Nice!

Note that this doesn't test installing as a non-root user. Definitely better than nothing though!

File: scripts/install_linux.sh [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+	exit 1
+fi
+
+DOWNLOAD_URL=$(curl -s ${RELEASE_URL} | grep "browser_download_url.*docker-linux-amd64" | cut -d : -f 2,3)
+
+# Check if the ACI CLI is already installed
+if [ $(is_new_cli "docker") -eq 1 ]; then
+	echo "You already have the Docker ACI Integration CLI installed, overriding with latest version"
+	download_dir=$($sh_c 'mktemp -d')
+	$sh_c "${download_cmd} ${download_dir}/docker-aci ${DOWNLOAD_URL}"
```
Comment (chris-crone) [inline scripts/install_linux.sh:97]: Would be good to factor this with the download code [lower down](https://github.com/docker/aci-integration-beta/pull/17/files#diff-65f46e4d078e4e0901b0680970a6d659R146-R152). Can be done in a follow up.

File: scripts/install_linux.sh [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+	$sudo_sh_c "install -m 775 ${download_dir}/docker-aci /usr/local/bin/docker"
```
Comment (chris-crone) [inline scripts/install_linux.sh:98]: We should check that a version of the CLI is present here and fail if it isn't. It's possible they manually installed to another location and then tried to update with the script.

Comment (gtardif) [reply scripts/install_linux.sh]: Fixed, checking $(is_new_cli "/usr/local/bin/docker").

**Pull request merged**